<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewInstrument</class>
 <widget class="QDialog" name="DialogNewInstrument">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建仪器</string>
  </property>
  <widget class="QComboBox" name="comboBoxIngestor">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据前摄器：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据处理器：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProcessor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>命令适配器：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>数据导出器：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDumper">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCommander">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>仪器描述器：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDescriptor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>仪器名字：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInstrumentName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogNewInstrument</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
